middleware/logging/writer.go

```diff
@@ -54,3 +57,14 @@ func (w *responseWriter) Flush() {
 	flusher := w.httpWriter.(http.Flusher)
 	flusher.Flush()
 }
+
+// Hijack implements the http.Hijack interface.
+// This is used by the server-sent events implementation to keep the long-running (idle) connection open.
+func (w *responseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
+	hj, ok := w.httpWriter.(http.Hijacker)
+	if !ok {
+		return nil, nil, errors.New("hijacker interface not supported")
```

Comment:
'hijacker interface' sounds irritating, probably write it explicitly as "http.Hijacker interface not supported"?